packages/vue-dot/src/elements/DataList/tests/data/dataList.ts

```diff
@@ -0,0 +1,16 @@
+import { IDataListItem } from './../../types';
+
+export const list: IDataListItem[] = [
```

Comment:
```suggestion
export const dataLlist: IDataListItem[] = [
```
`list` c'est un peu trop générique !